src/mlio-py/mlio/__init__.py

```diff
@@ -67,8 +70,10 @@
     'CorruptHeaderError',
     'CorruptRecordError',
     'CsvReader',
+    'CsvReaderParams',
     'DataReader',
     'DataReaderError',
+    'DataReaderParams'
```

Comment:
You missed the comma at the end.

Reply:
Good catch. I'm not sure how this was able to build...